### PR TITLE
Various fixes/features

### PR DESCRIPTION
**Pull request by SeraphJACK**
- Add last_modified field in lyric response
- Add source field in lyric response
- Add last_modified field in playlist response
- Allow null PlaylistCover which tells the client cover should be the same as the first track of the playlist

### CODE REVIEW

File: src/06.anniv/03.playlist.md [hunk 1]
```diff
@@ -85,10 +86,11 @@
         "track_id": 1,
         "title": "ハルノヘッドフォン",
         "artist": "春日部ハル（篠田みなみ）",
-        "album_title": "t7s Longing for summer",
+        "album_title": "t7s Longing for summer"
       }
     }
-  ]
+  ],
+  "last_modified": 1145141919810
```
Comment (Yesterday17) [inline src/06.anniv/03.playlist.md:93]: 好臭啊（x

File: src/06.anniv/11.lyric.md [hunk 2]
```diff
@@ -61,10 +65,11 @@ interface LyricLanguage {
 | `track_id` | `number`                         | 请求歌词的单曲 `ID` |
 | `type`     | <code>"text" &#124; "lrc"</code> | 修改后歌词的类型    |
 | `lang`     | `LanguageCode`                   | 修改歌词的语言      |
+| `source`   | `string`                         | 歌词来源            |
```
Comment (Yesterday17) [inline src/06.anniv/11.lyric.md:68]: 加一个歌词最近更新？

File: src/06.anniv/11.lyric.md [hunk 3]
```diff
@@ -26,11 +26,15 @@ interface LyricLanguage {
   // 歌词语言
   language: LanguageCode;
   // 歌词类型
-  type: "text" | "lrc";
+  type: "text" | "lrc" | "karaoke";
```
Comment (Yesterday17) [inline src/06.anniv/11.lyric.md]: karaoke的格式？

**Review by Yesterday17 — CHANGES_REQUESTED** [review]

File: src/06.anniv/11.lyric.md (hunk 3 above)
Comment (SeraphJACK) [reply src/06.anniv/11.lyric.md]: karaoke的格式？

File: src/06.anniv/11.lyric.md (hunk 2 above)
Comment (SeraphJACK) [reply src/06.anniv/11.lyric.md:68]: 这不是 POST 的表单吗

Comment (SeraphJACK) [reply src/06.anniv/11.lyric.md:68]: 返回里面有的，修改表单里不应该有

File: src/06.anniv/11.lyric.md (hunk 3 above)
Comment (Yesterday17) [reply src/06.anniv/11.lyric.md]: 那我建议先不加 karaoke，后续 pr 再定（

File: src/06.anniv/03.playlist.md [hunk 4]
```diff
@@ -36,15 +36,16 @@
...
+| 参数名          | 类型                    | 详情                                                                  |
+| --------------- | ----------------------- | --------------------------------------------------------------------- |
+| `id`            | `string`                | 播放列表标识符                                                        |
+| `name`          | `string`                | 播放列表名称                                                          |
+| `description`   | `string`                | 播放列表说明                                                          |
+| `owner`         | `string`                | 播放列表拥有者 `ID`                                                   |
+| `is_public`     | `boolean`               | 是否公开                                                              |
+| `items`         | `(PlaylistItem & Id)[]` | 播放列表中的内容                                                      |
+| `cover`         | `PlaylistCover`         | 播放列表封面，为 `null` 时客户端应使用该 Playlist 的第一个 Track 封面 |
+| `last_modified` | `number`                | 播放列表最后修改时间戳                                                |
```
Comment (Yesterday17) [inline src/06.anniv/03.playlist.md]: 单位为秒

File: src/06.anniv/11.lyric.md [hunk 5]
```diff
@@ -61,10 +65,11 @@ interface LyricLanguage {
...
 | `data`     | `string`                         | 修改后的歌词        |
 
 #### 错误代码
 
-| 错误代码 | 详情                                                      |
-| -------- | --------------------------------------------------------- |
-| `901000` | 歌词修改无法写入，可能的原因有：修改 lrc 歌词为 text 歌词 |
+| 错误代码 | 详情                                                                                   |
+| -------- | -------------------------------------------------------------------------------------- |
+| `901000` | 歌词修改无法写入，可能的原因有：修改 lrc 歌词为 text 歌词、修改 karaoke 歌词为其他歌词 |
```
Comment (Yesterday17) [inline src/06.anniv/11.lyric.md]: 这里还有 karaoke（

**Review by Yesterday17 — CHANGES_REQUESTED** [review]

**Review by Yesterday17 — APPROVED** [review]

**Pull request merged**